jbpm-wb-integration/jbpm-wb-integration-client/src/test/java/org/jbpm/workbench/wi/client/workitem/ServiceTaskUploadFormPresenterTest.java

```diff
@@ -60,14 +64,26 @@
     @Mock
     private SyncBeanManager iocManager;
 
+    private List<String> createdList = Collections.singletonList("Created");
+    private List<String> updatedList = Collections.singletonList("Updated");
+
+    private String addTaskSuccess = "Service tasks successfully added -";
+    private String updateTaskSuccess = "Service tasks successfully updated -";
+
     @Before
     public void before() {
-        when(serviceTaskService.addServiceTasks(any())).thenReturn(new ArrayList<>());
+        Map<String, List<String>> resultMap = new HashMap<String, List<String>>();
+        resultMap.put(ServiceTaskUploadFormPresenter.CREATED, createdList);
+        resultMap.put(ServiceTaskUploadFormPresenter.SKIPPED, updatedList);
+        when(serviceTaskService.addServiceTasks(any())).thenReturn(resultMap);
         presenter = spy(new ServiceTaskUploadFormPresenter(view,
                                                            new CallerMock<ServiceTaskService>(serviceTaskService),
                                                            notificationEvent,
                                                            iocManager));
         presenter.init();
+
+        when(view.getSuccessInstallMessage()).thenReturn(addTaskSuccess);
+        when(view.getSkippedMessage()).thenReturn(updateTaskSuccess);
```

Comment:
there's no update anymore, right? we either create or skip (At least that's what I see in impl)